pkg/render/apiserver.go

```diff
@@ -344,24 +396,19 @@ func (c *apiServerComponent) apiServerServiceAccount() *corev1.ServiceAccount {
 	return &corev1.ServiceAccount{
 		TypeMeta: metav1.TypeMeta{Kind: "ServiceAccount", APIVersion: "v1"},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "tigera-apiserver",
-			Namespace: APIServerNamespace,
+			Name:      serviceAccountName(c.installation.Variant),
+			Namespace: rmeta.APIServerNamespace(c.installation.Variant),
 		},
 	}
 }
 
-// apiServiceAccountClusterRole creates a clusterrole that gives permissions to access backing CRDs and
-// k8s networkpolicies.
-func (c *apiServerComponent) apiServiceAccountClusterRole() *rbacv1.ClusterRole {
+// calicoCustomResourcesClusterRole creates a clusterrole that gives permissions to access backing CRDs and k8s networkpolicies.
+func (c *apiServerComponent) calicoCustomResourcesClusterRole() *rbacv1.ClusterRole {
 	rules := []rbacv1.PolicyRule{
 		{
-			APIGroups: []string{
-				"extensions",
```

Comment:
extensions is long-since deprecated, and all of the resources within it have been moved to other API groups (note this is different from apiextensions)